test/analysis-tests.ts

```diff
@@ -23,10 +23,12 @@
 // POSSIBILITY OF SUCH DAMAGE.
 
 import {LLVMmcaTool} from '../lib/compilers/llvm-mca';
+import {CompilerInfo} from '../types/compiler.interfaces';
+import {LanguageKey} from '../types/languages.interfaces';
 
 import {makeCompilationEnvironment} from './utils';
 
-const languages = {
+const languages: {analysis: {id: LanguageKey}} = {
     analysis: {id: 'analysis'},
 };
```

Comment:
We can have TS infer its strictest type automatically
```suggestion
const languages = {
    analysis: {id: 'analysis'},
} as const;
```